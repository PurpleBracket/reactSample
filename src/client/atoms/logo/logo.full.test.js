/* global describe test expect */
import React from 'react'
import { shallow } from 'enzyme'
import LogoFull from './logo.full'

describe('LogoFull', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<LogoFull />)
    expect(wrapper).toMatchSnapshot()
  })
})
