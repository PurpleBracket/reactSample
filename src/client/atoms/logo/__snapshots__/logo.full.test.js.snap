// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoFull should match snapshot 1`] = `
<img
  alt="Treasure"
  src={Object {}}
/>
`;
